templates/base-secrets.json

```diff
@@ -75,4 +75,8 @@
   "id": "vpc-1234567890xx",
   "name": "some-other-vpc"
 }
+],
+"aws_custom_tags": [
+  {"key":  "project", "value":  "foo"},
+  {"key":  "department", "value":  "bar"}
```

Comment:
Do the new cluster's S3 buckets always get these tags?
https://github.com/harvard-dce/mh-opsworks/blob/8d5e83bc22c6b033bcd184efd2ec1bc9b9b34018/lib/cluster/s3_bucket.rb#L14-L38

Will adding custom project & department tags overwrite the official project & department tags? Does it even make sense to allow custom tagging?

Reply:
The `base-secrets.json` that comes in the code repo is just an example/template. Our base-secrets.json file is in the cluster configs bucket of each account. The default tags for new clusters are:

```
"aws_custom_tags": [
  {"key":  "project", "value":  "MH"},
  {"key":  "department", "value":  "DE"},
  {"key":  "product", "value":  "opencast"}
],
```

For the 2nd question, i'm not sure what you meant. The use of "custom" in the name may be confusing. Perhaps I should take the opportunity to rename it to just "aws_tags"?

Reply:
If I have this in my cluster config, when creating a cluster, will it wipe out the official project, department, product tag?
"aws_custom_tags": [
  {"key":  "project", "value":  "MH"},
  {"key":  "department", "value":  "DE"},
  {"key":  "product", "value":  "opencast"}
],

Reply:
i don't know what you mean by "official". There's only these tags.

Reply:
@lbjay Ah, the "foo" and "bar" get replaced by the secrets file values, right?